lib/redisWrapper.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var redisWrapper = module.exports = {};
+var Q = require('q');
+var redis = require('redis');
+var _ = require('lodash');
+var redisClient;
+var separator = ':';
+var wildcard = '*';
+
+redisWrapper.initialise = function(serverConfig) {
+  var deferred = Q.defer();
+  if (_.isEmpty(serverConfig)) {
+    deferred.reject('no serverConfig passed');
+  } else {
+    redisClient = redis.createClient(serverConfig);
+    redisClient.on('ready', function () {
+      deferred.resolve();
+    });
+    // if there was error connecting to redis server
+    redisClient.on('error', function () {
```

Comment:
I do not get any error locally  where do you get this and tests are passing on the PR as well